src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -15,4 +15,34 @@
   <Import Condition="'$(_packageTargetOSGroup)' != ''"  Project="$(MSBuildThisFileDirectory)runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props" />
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
+
+
+  <PropertyGroup>
+    <!-- We're publishing crossgen2 as framework dependent application in the runtime package for now since we only have
+        two simple scenarios we want to support in the short term: win_x64 to win_x64 and linux_x64 to linux_x64 compilations.
+        Once we have more complex targets, especially cross-platform and cross-architecture, crossgen2 should move to its own
+        package, and become a self-contained package. -->
+    <Crossgen2RuntimeConfigFile>$(ArtifactsObjDir)crossgen2\crossgen2.runtimeconfig.json</Crossgen2RuntimeConfigFile>
+    <Crossgen2RuntimeConfigContents>
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp3.0",
+    "rollForward": "major",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "3.0.0"
```

Comment:
Maybe this should be 5 with a rollForward=minor instead...

Reply:
version 5.0 with a rollForward=minor seems to be broken... I tested using 5.0.0-alpha1.19507.3, but got the error that no runtime was found. It works only if rollForward == major.
Maybe a CLI bug? 
cc @nguerrera

Reply:
I would need to see a repro. The CLI is not involved if you are hand crafting a runtime config.

Reply:
@nguerrera I IM'd you the repro. I couldn't upload it here as an attachment for some reason

Reply:
I'll use a version of 5.0 and rollforward=major for now. We need to handcraft the runtimeconfig.json to package crossgen2 as a FD application. We publish it today as a self-contained app, and the runtimeconfig for it would be empty